### PR TITLE
Fix error prone name clash error

### PR DESCRIPTION
**Pull request by etanshaul**
image

### CODE REVIEW

File: google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/NamedResourceInspectionTest.java [hunk 1]
```diff
@@ -58,7 +59,7 @@ public void testMethodsWithUnnamedResourcesEmptyString() {
 
   /**
    * Tests that the NamedResourceInspection's quick fix flagged with
-   * {@link NamedResourceInspection.Error.DUPLICATE_PARAMETER} for an @Named annotation updates
+   * {@link NamedResourceError.DUPLICATE_PARAMETER} for an @Named annotation updates
```
Comment (etanshaul) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/NamedResourceInspectionTest.java]: need to fix these

File: google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/NamedResourceInspectionTest.java [hunk 2]
```diff
@@ -58,7 +59,7 @@ public void testMethodsWithUnnamedResourcesEmptyString() {
 
   /**
    * Tests that the NamedResourceInspection's quick fix flagged with
-   * {@link NamedResourceInspection.Error.DUPLICATE_PARAMETER} for an @Named annotation updates
+   * {@link NamedResourceError#DUPLICATE_PARAMETER} for an @Named annotation updates
```
Comment (nkibler) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/NamedResourceInspectionTest.java]: nit: Do you mind running the formatter on this javadoc (and the ones below) while you're here?

Comment (etanshaul) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/NamedResourceInspectionTest.java]: done, I think

Comment (nkibler) [inline google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/validation/NamedResourceInspectionTest.java]: Yep that's right now, thanks.

**Pull request merged**